src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -421,4 +424,17 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 		}
 		this._editor.setHeightToScrollHeight(false, isCollapsed);
 	}
+
+	private onCellModeChanged(isEditMode: boolean): void {
+		let ownerDocument = this._editor.getContainer().ownerDocument;
+		if ((this.cellModel.id === this._activeCellId || this._activeCellId === '') && this._editor.getContainer().offsetParent && ownerDocument && ownerDocument.hasFocus()) {
```

Comment:
comment explaining what you're doing here and what these checks are for

Reply:
Removed this checks since they are not necessary for this. Thought we needed this because we have this checks for scrolling to code cells when they are run.